spec.emu

```diff
@@ -62,23 +62,56 @@ contributors: Google, Ecma International
         The IsValidCodeForDateTimeField abstract operation verifies that the given _code_ is valid.
```

Comment:
In a follow-up, maybe move the IsValidCodeFor* methods to the bottom of the spec file, maybe in their own section, because otherwise if we add more sections in the future, we have to add them to the bottom in order to keep section numbers stable.